lib/fortenet/request.rb

```diff
@@ -1,55 +1,41 @@
-require 'httparty'
-require 'json'
-
 module Fortenet
   class Request
     include HTTParty
+
     format :json
     headers 'Accept' => 'application/json'
 
-    def get(path, options={})
-      set_auth(options)
-      @response = self.class.get(Fortenet.endpoint + path, options)
-    end
+    attr_reader :response
 
-    def post(path, options={})
-      set_auth(options)
-      @response = self.class.post(Fortenet.endpoint + path, options)
+    def get(path, **options)
+      self.response = self.class.get(Fortenet.endpoint + path, set_auth(options))
     end
 
-    def put(path, options={})
-      set_auth(options)
-      @response = self.class.put(Fortenet.endpoint + path, options)
+    def post(path, **options)
+      self.response = self.class.post(Fortenet.endpoint + path, set_auth(options))
     end
 
-    def delete(path, options={})
-      set_auth(options)
-      @response = self.class.delete(Fortenet.endpoint + path, options)
+    def put(path, **options)
+      self.response = self.class.put(Fortenet.endpoint + path, set_auth(options))
     end
 
-    def response
-      @response
+    def delete(path, **options)
+      self.response = self.class.delete(Fortenet.endpoint + path, set_auth(options))
     end
 
     def parsed_response
-      self.response.parsed_response if self.response.present?
+      return if response.nil?
+
+      JSON.parse response.body, symbolize_names: true
```

Comment:
because i wanted symbolized keys but `.with_indifferent_access` is a rails dependency and didn't want to have to call that every time when using this and httpartys parsed_reponse didn't seem to have any options for symbolizing keys either